modules/gstama/polyacleanup/main.nf

```diff
@@ -0,0 +1,34 @@
+process GSTAMA_POLYACLEANUP {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gs-tama=1.0.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gs-tama:1.0.3--hdfd78af_0':
+        'quay.io/biocontainers/gs-tama:1.0.3--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*_tama.fa")                   , emit: fasta
+    tuple val(meta), path("*_tama_polya_flnc_report.txt"), emit: report
+    tuple val(meta), path("*_tama_tails.fa")             , emit: tails
+    path "versions.yml"                                  , emit: versions
+
```

Comment:
```suggestion

    when:
    task.ext.when == null || task.ext.when
    
```

Reply:
@sateeshperi Thanks for pointing the error. I updated my nf-core and fixed it.